DOC: guide developer to fix registry checks warnings/errors (#284)
aiidateam/aiida-registry: README.md

since this both "reserves" the name of your plugin and informs the developer
community of your ongoing work.

By default, the list of plugins is now sorted by the latest release, so plugins that are under active development automatically bubble up to the top.

> The release date is determined by the date of the latest [PyPI](https://pypi.org/) release, so try to release to [PyPI](https://pypi.org/) whenever you make a new release of your plugin.

Reviewer: ```suggestion
The release date is determined by the date of the latest [PyPI](https://pypi.org/) release. Plugins not released to PyPI will have no release date.
```

Over time, we've moved closer and closer to adopting the [development status trove classifer](https://pypi.org/classifiers/), so we now suggest to just use those in your `setup.json`/`setup.cfg`/... file of your plugin.

If no development status is specified, the status will default to 'planning'.

> ## How to fix registry warnings and errors

Reviewer: I get that installing the registry locally is useful for debugging, but I think our goal should be that plugin developers do _not_ have to do this.

Once https://github.com/aiidateam/aiida-registry/issues/272 is fixed, I think the default suggestion should be to open a PR and look at the warnings in the build.
Is there an intermediate suggestion we can write in the README until we have that capability?

As for the rest of this section, in order to keep the README short, I suggest moving it in a separate markdown document 'debugging.md' and link to it from the README.

Author: > I get that installing the registry locally is useful for debugging, but I think our goal should be that plugin developers do not have to do this.

I agree. It is better to move to the repository wiki and link from README.

You may want to check that your plugin is correctly registered before making the changes to your plugin repository.
To do so, you can run the registry check locally.

Clone this repository and install the dependencies:
```bash
git clone https://github.com/aiidateam/aiida-registry.git
pip install aiida-registry
```

Fetch the metadata and run installation test or your plugin
```bash
aiida-registry fetch-metadata <plugin-name>

> aiida-registry test-install

Reviewer: this runs it just for the one plugin, is that correct?

Author: This support fetches metadata for many plugins. The `aiida-registry test-install` will then install only plugins from the list.
